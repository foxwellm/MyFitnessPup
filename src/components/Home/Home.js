import React from 'react'
import { NavLink } from 'react-router-dom'

export const Home = (props) => {
  return (
    <div className='home-container'>
    {
        props.isAbout && <div className='shaddow'></div>
    }
      <img className='home-img' alt='dog infront of mountains' src={require("../../assets/mountains.jpg")} />
      {
        props.isAbout && <div className='home-card'>
          <NavLink to='/' className='home-btn'><img className='close-img' alt='close button' src={require("../../assets/cross-mark.png")} /></NavLink> 
          <p>MyFitnessPup is here to help you find the perfect dog to take hiking/running with you. Learn about the breeds best suited for high activity, then find dogs that are in shelters right by you to take on your next adventure.</p></div>
      }
    </div>
  )
}

export default Home