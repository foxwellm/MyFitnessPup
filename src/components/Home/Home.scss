.home-container {
  left: 0;
  top: 80px;
  height: 100%;
  width: 100%;
  position: fixed;
}

.home-card {
  background-color: $primary-light;
  font-family: $font-secondary;
  padding: 20px;
  border-radius: 5px;
  left: 30vw;
  top: 30vh; 
  width: 40vw;
  height: 40vh;
  min-height: 300px;
  min-width: 600px;
  position: fixed;
  color: white;
  font-size: 35px;
  z-index: 2;
}

.shaddow {
  position: absolute;
  top: 0;
  left: 0;
  height: 100%;
  width: 100%;
  z-index: 1;
  background-color: black;
  opacity: .3;
}

.home-img {
  width: 100%;
  height: 100%;
}

.close-img {
  float: right;
  width: 30px;
  height: 30px;
  &:hover {
    transform: scale(1.05)
  }
}