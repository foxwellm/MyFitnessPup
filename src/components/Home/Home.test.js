import React from 'react'
import { shallow } from 'enzyme'
import Home from './Home'

describe('Home', () => {
  let wrapper
  it('should have match the correct snapshot', () => {
    wrapper = shallow(<Home />)
    expect(wrapper).toMatchSnapshot()
  })
})