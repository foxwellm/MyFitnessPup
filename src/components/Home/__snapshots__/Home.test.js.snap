// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should have match the correct snapshot 1`] = `
<div
  className="home-container"
>
  <img
    alt="dog infront of mountains"
    className="home-img"
    src="mountains.jpg"
  />
</div>
`;
